django_fido/admin/authenticator.py

```diff
@@ -0,0 +1,108 @@
+"""Admin for django_fido authenticator."""
+from __future__ import unicode_literals
+
+from django import forms
+from django.conf.urls import url
+from django.contrib import admin
+from django.contrib.auth import get_user_model
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.http import Http404, HttpResponse, HttpResponseRedirect
+from django.urls import reverse, reverse_lazy
+from django.utils.translation import ugettext_lazy as _
+
+from django_fido.forms import Fido2RegistrationForm
+from django_fido.models import Authenticator
+from django_fido.views import Fido2RegistrationRequestView, Fido2RegistrationView
+
+
+class Fido2RegistrationAdminForm(Fido2RegistrationForm):
+    """Registration form with user selection."""
+
+    user = forms.ModelChoiceField(
+        queryset=get_user_model().objects.all(),
+        error_messages={'required': _("Operation wasn't completed.")},
+    )
+
+    field_order = ('user', 'label')
+
+
+class Fido2RegistrationRequestAdminView(PermissionRequiredMixin, Fido2RegistrationRequestView):
+    """Registration request view with user selection."""
+
+    permission_required = 'django_fido.add_authenticator'
+
+    def get_user(self):
+        """Get user based on POST request."""
+        user_id = int(self.request.POST.get('user'))
```

Comment:
`user_pk` doesn't have to be an integer.

Reply:
Fixed.